simapp/v2/export.go

```diff
@@ -1,10 +1,48 @@
 package simapp
 
 import (
+	"context"
+	"cosmossdk.io/x/staking"
+
+	cmtproto "github.com/cometbft/cometbft/api/cometbft/types/v1"
+	cmttypes "github.com/cometbft/cometbft/types"
 	servertypes "github.com/cosmos/cosmos-sdk/server/types"
 )
 
-// ExportAppStateAndValidators exports the state of the application for a genesis file.
+// ExportAppStateAndValidators exports the state of the application for a genesis
+// file.
 func (app *SimApp[T]) ExportAppStateAndValidators(forZeroHeight bool, jailAllowedAddrs, modulesToExport []string) (servertypes.ExportedApp, error) {
```

Comment:
Can we have another function signature here? And maybe have genutil have a `genutil.CommandsV2` instead?
We shouldn't depend on `"github.com/cosmos/cosmos-sdk/server/types"` in a v2 app